### DISCUSSION TITLE
What is the concurrency for online and offline transcription across different tiers?

### DISCUSSION THREAD

**Discussion by MohitMayankOP** [deepgram/community#314] [General help] [answered]
Hey, I just started exploring Deepgram, but before we can consider it seriously there is the question of scalability - what is the concurrency (i.e. parallel analysis) DeepGram supports for online and offline transcription in Free and other tiers?
Any help will be appreciated. Thanks!

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @MohitMayankOP, Deepgram offers robust concurrency across all tiers for all our in-house models, including our best-in-class Nova model. We do rate-limit Whisper usage as it is a much more compute-intensive model, so I highly encourage you to get started with Nova for English and Spanish, and Base or Enhanced for other languages. See our full docs of [models by language](https://developers.deepgram.com/docs/languages-overview).
